ui/src/LandingPage.js

```diff
@@ -40,11 +40,15 @@ class LandingPage extends Component {
     templateValidationError: false,
     newSurveyHeaderRow: true,
     newSurveySampleSeparator: ",",
+    triggerID: "366ce7da-f885-493d-b933-a3b74d583a06",
+    nextFulfilmentTriggerDateTime: "1970-01-01T00:00:00.000Z",
+    configureNextTriggerDisplayed: false,
   };
 
   componentDidMount() {
     this.getAuthorisedActivities(); // Only need to do this once; don't refresh it repeatedly as it changes infrequently
     this.refreshDataFromBackend();
+    this.getFulfilmentTrigger();
```

Comment:
a nice small/easy/simple enhancement might be to call this every time that the user opens the dialog box, so that it refreshes the data from the backend